release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -1135,6 +1140,13 @@ This release deprecates the `lastTriggeredImageID` in the `ImageChangeTrigger` o
 
 Therefore, update scripts and jobs that inspect `buildConfig.spec.triggers[i].imageChange.lastTriggeredImageID` accordingly. (link:https://issues.redhat.com/browse/BUILD-213[*BUILD-213*])
 
+[id="ocp-4-8-builds-jenkins-operator"]
+==== The Jenkins Operator is deprecated
+
+This release deprecates the Jenkins Operator. A future version of {product-title} will remove support for the Jenkins Operator and ignore it. Then, <describe of effects of removal>.
```

Comment:
@siamaksade I think we need to clarify this a bit:
```suggestion
This release deprecates the Jenkins Operator. A future version of {product-title} will remove the Jenkins Operator from OperatorHub. Then, upgrades for the Jenkins Operator will no longer be available and the operator will not be supported.
```

Reply:
Updated.